nss-day-cohort-44/nutshell-whistling-wraiths
# Dd weather

# Description
Adds a show weather button to all events that displays the weather.  Should also have fixed all issues with deleting/creating/editing events.
Fixes # (issue)
## Type of change
Please delete options that are not relevant.

# Testing Instructions
Please make sure weather displays for the current day for all events. Make sure you have a Settings.js that is not tracked (use .gitignore) that exports the appropriate API keys.
# Checklist:
- [x ] My code follows the style guidelines of this project
- [ x] I have performed a self-review of my own code
- [ x] I have commented my code, particularly in hard-to-understand areas
- [ x] My changes generate no new warnings or errors
- [ x] I have added test instructions that prove my fix is effective or that my feature works
